### ISSUE TITLE
Error: Cannot find module 'tar-stream'

### ISSUE THREAD

**Issue by pimterry** [resin-io/resin-cli#851]
https://sentry.io/resinio/cli/issues/518175738/

```
Error: Cannot find module 'tar-stream'
  File "module.js", line 469, in Function.Module._resolveFilename
  File "module.js", line 417, in Function.Module._load
  File "module.js", line 497, in Module.require
  File "internal/module.js", line 20, in require
  File "/opt/local/lib/node_modules/resin-cli/build/utils/compose.js", line 115, in tarDirectory
    tar = require('tar-stream');
...
(12 additional frame(s) were not displayed)

Error: Cannot find module 'tar-stream'
```

Looks like we're depending on this transitively, but not explicitly, which might be messing something up somewhere.

**Comment by pimterry** [comment]
Fixed in v7.2.4

**Comment by mohit-aicg** [comment]
Still getting this error in angular 12
